addon-test-support/ilios-common/page-objects/session.js

```diff
@@ -118,6 +118,10 @@ export default create({
       set: fillable('select'),
       save: clickable('.done'),
       hasError: isVisible('.validation-error-message')
+    },
+    lastUpdated: {
```

Comment:
I'm not sure why this one doesn't match `lastUpdated: text('.slast-update')` above.

Reply:
good catch. fixed.